src/pages/patterns/tables/basic-formatting.mdx

```diff
@@ -113,38 +73,134 @@ with some exceptions for quantitative data.
 
 ![Cell Alignment](figma:patterns-tables-basic-formatting-cell-alignment.png)
 
-<table>
-  <thead>
-    <tr>
-      <th width="70%">Cell format</th>
-      <th>Alignment</th>
-    </tr>
-  </thead>
-  <tbody>
-    <tr>
-      <td>1. Text</td>
-      <td>Left align</td>
-    </tr>
-    <tr>
-      <td>2. Categorical data that uses numbers (for example, dates, phone numbers)</td>
-      <td>Left align</td>
-    </tr>
-    <tr>
-      <td>3. Quantitative data</td>
-      <td>Right align</td>
-    </tr>
-    <tr>
-      <td>
-        4. <a href="/components/tags">Tags</a> or <a href="/components/avatar">Avatars</a>
-      </td>
-      <td>Left align</td>
-    </tr>
-    <tr>
-      <td>5. Icons</td>
-      <td>Center align</td>
-    </tr>
-  </tbody>
-</table>
+| <b>Cellc content</b>                                                                                        | <b>Alignment</b> |
+| ----------------------------------------------------------------------------------------------------------- | ---------------- |
+| 1. Text                                                                                                     | Left align       |
+| 2. Categorical data that uses numbers (for example, [dates](/content/grammar#date-and-time), phone numbers) | Left align       |
+| 3. Quantitative data                                                                                        | Right align      |
+| 4. [Tags](/components/tags) or [Avatars](/components/avatar)                                                | Left align       |
+| 5. Icons                                                                                                    | Center align     |
+
+## Avatars, icons, and tags
+
+### Avatars
+
+[Avatars](/components/avatar) help represent a person or brand in tables. They can display icons or images.
+
+### Size
+
+Avatar sizes for tables include default and small.
+
+![Small avatars in a Default table](figma:patterns-tables-basic-formatting-small-avatars-default-table.png)
```

Comment:
Something I noticed in this image is the size blocks are the same color, but communicate different dimensions/values (10px vs 12px). Can one be red and another blue, for example?

Also, I am slightly color blind so if they're actually different already, probably even more important to change them 😅

Reply:
oo good point. from what i understand, the red shades are just to indicate spacing in general, not necessarily specific sizes. i think it's worth revisiting our website/documentation annotations to address this!

Reply:
No worries. If it already matches everything else, let's not change it here until it can be addressed holistically later. :)